docs/feature_split_keyboard.md

```diff
@@ -79,6 +79,15 @@ If you're using a custom transport (communication method), then you will also ne
 SPLIT_TRANSPORT = custom
 ```
 
+### Layout Macro
+When configuring a split keyboard, note the matrix rows and pins are doubled, thus QMK views the layout as follows, where the numbers represent `row, col`:
+
+IMG1
+
+IMG2
+
+Take care to define your layout in `info.json` accordingly.
```

Comment:
```suggestion

Configuring your layout in a split keyboard works slightly differently to a non-split keyboard. Take for example, the following layout. The top left numbers refer to the matrix row and column, and the bottom right are the order of the keys in the layout:

![Physical layout](https://i.imgur.com/QeY6kMQ.png)

Since the matrix scanning procedure operates on entire rows, it first populates the left half's rows, then the right half's. Thus, the matrix as QMK views it has double the rows instead of double the columns:

![Matrix](https://i.imgur.com/4wjJzBU.png)
```

Reply:
This is a tremendous improvement! Thank you for taking the time to do this!